examples/elm/index.html

```diff
@@ -1,34 +1,31 @@
-<!doctype html>
-<html lang="en" data-framework="elm">
+<!DOCTYPE HTML>
+<html>
+
 <head>
-	<meta charset="utf-8">
-	<title>Elm • TodoMVC</title>
-	<link rel="stylesheet" href="node_modules/todomvc-common/base.css">
-	<link rel="stylesheet" href="node_modules/todomvc-app-css/index.css">
-	<style>
-		body {
-			width: auto;
-		}
-		.todomvc-wrapper {
-			visibility: visible !important;
-		}
-	</style>
+  <meta charset="UTF-8">
+  <title>Elm • TodoMVC</title>
+    <link rel="stylesheet" href="node_modules/todomvc-common/base.css">
+    <link rel="stylesheet" href="node_modules/todomvc-app-css/index.css">
+    <style>
+      body {
+        width: auto;
+      }
+      .todomvc-wrapper {
+        visibility: visible !important;
+      }
+    </style>
+  <script type="text/javascript" src="build/elm.js"></script>
 </head>
 
 <body>
-<script src="build/elm.js"></script>
-<script>
-	(function () {
-		var result = localStorage.getItem('elm-todo-model');
-		var savedModel = result ? JSON.parse(result) : null;
-
-		var todomvc = Elm.Todo.fullscreen(savedModel);
+</body>
 
-		todomvc.ports.save.subscribe(function (model) {
-			localStorage.setItem('elm-todo-model', JSON.stringify(model));
-		});
-	}());
+<script type="text/javascript">
```

Comment:
Can you wrap the code in a closure here to not leak the variables to the global scope?

Reply:
Can you show how you want me to do this?